package.json

```diff
@@ -30,10 +30,10 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "semistandard": "^16.0.1",
-    "ts-loader": "9",
+    "ts-loader": "9.5",
     "tslint": "^5.16.0",
-    "typescript": "^3.9.10",
-    "webpack": "5",
+    "typescript": "^4.9.0",
```

Comment:
The TS upgrade is a little worrisome. Do we have any assurance that everything is still working fine at runtime?

Reply:
It looks ok in AutoTest, and the 4.0 release notes say there are no breaking changes and updating should be invisible to the majority of projects.